internal/filter/filter.go

```diff
@@ -0,0 +1,41 @@
+package filter
+
+import (
+	"sync/atomic"
+
+	"github.com/Azure/azure-storage-fuse/v2/internal"
+)
+
+var GlbFilterArr [][]Filter //it will store the fliters, outer array splitted by ||, inner array splitted by &&
```

Comment:
As part of design we shall move these filters to be a class so that there can be multiple instances of such filters across the blobfuse code. Current design of having it as a global variable restricts that only one such filter can exists in scope of blobfuse instance.